moveit_ros/moveit_servo/src/servo_node.cpp

```diff
@@ -38,9 +38,14 @@
  *
  */
 
-#include <moveit_servo/servo_node.hpp>
+#if __has_include(<urdf/realtime_tools/realtime_helpers.hpp.hpp>)
```

Comment:
gotta remove the urdf part!

Reply:
attention to detail is not my strength... lol

Reply:
```suggestion
#if __has_include(<realtime_tools/realtime_helpers.hpp.hpp>)